ftag/sample.py

```diff
@@ -23,14 +23,18 @@ def __post_init__(self) -> None:
             raise FileNotFoundError(f"The following files do not exist: {missing}")
 
     @property
-    def path(self) -> Path:
+    def path(self) -> tuple[Path]:
+        pattern_tuple = self.pattern if isinstance(self.pattern, tuple) else (self.pattern,)
```

Comment:
Might be worth also checking for list alongside tuple here, no reason not to support both

Reply:
I'm not an expert with function return annotations, but from what I found online (https://peps.python.org/pep-0585/), `tuple`, `list` and others are now generic types, so I don't think it's necessary?

Reply:
generic in this context means the type accepts a specifier, e.g. `list[str]`. `list` and `tuple` are still consindered different to type checkers. you can do `list | tuple` or use `typing.Sequence` to indicate both are allowed.

Reply:
Sorry to clarify I am talking about the `isinstance(self.pattern, tuple)`, it should be `isinstance(self.pattern, list | tuple)` to make sure things don't break if someone instantiates the class with a list.

Reply:
ooooh my bad, I was looking at the wrong line 🙃Yes, will fix right now